ui/templates/base.html

```diff
@@ -37,5 +37,6 @@
     </ul>
   {% endif %}
   {% block content %}{% endblock %}
+  <script src="https://cdn.sc.gl/videojs-hotkeys/latest/videojs.hotkeys.min.js"></script>
```

Comment:
Instead of including it this way, can you add it to `package.json` via:
```
docker-compose run watch yarn add videojs-hotkeys@^0.2.25
```
and then add the following to `static/lib/video.js`:
```
require("videojs-hotkeys")
```